### DISCUSSION TITLE
[General] joining discord channel

### DISCUSSION THREAD

**Discussion by Ryuminh** [zkSync-Community-Hub/zksync-developers#185] [General] [answered]
### Environment

Mainnet

### Provide a brief description of the functionality you're trying to implement.

hello i have some issue, i was in the discord channel and i don't know why i'm not in there anymore and i can't join it 

thanks to help me for this issue

astrohades.zksyncyujian.zk

this is my discord handle 

### What is the specific issue or error you're encountering?

discord 

### Can you share the error messages or logs you're receiving, if any?

discord

### Have you made any recent changes to the contract before encountering this issue?

discord

### Are there any external libraries or contracts that your contract interacts with?

discord

### Can you provide the relevant portions of your contract code where the issue is occurring?

discord

### Have you tried to isolate the problem, and if so, what were the results?

discord

### What steps have you already taken to try to resolve the issue?

discord

**Comment by dutterbutter** [comment]
@bxpana can you assist here please :)

**Comment by bxpana** [comment, marked as answer]
@Ryuminh sorry to hear you're having an issue with getting into the Discord. The Github Discussion page is for developer related questions, so we'd be happy to help you with your Discord issue at support@zksync.io. Send us an email with your Discord username, Discord ID number and we can help look into things. Talk to you there!

**Reply by Ryuminh** [reply]
hello i didnt had any news from the support, i sent a mail few days ago 

sincerelly

**Reply by bxpana** [reply]
What is your Discord name or what is a way to find your email?

**Reply by Ryuminh** [reply]
I sent with this mail : 0x7eef6f6affb8d30dbfc332e9f3065b27ae80d5e1@dmail.ai

that is my discord 

astrohades.zksyncyujian.zk

User ID : 472708612415029249

**Reply by bxpana** [reply]
Doesn't look like we received your email, but I do see why you're not able to join the server. You're username contains the name "zkSync" which is typically used by scammers on Discord. Once you remove that from your Discord username, let us know and you'll be able to join the server again.

**Reply by Ryuminh** [reply]
Hello 

I tried again but the link Said link is not valid or expired liké this

![Screenshot_2023-11-07-04-13-28-432_com discord](https://github.com/zkSync-Community-Hub/zkync-developers/assets/118870831/f24dd3fc-032b-450a-a240-651b1a1f590b)

**Reply by bxpana** [reply]
Ok, got things figured out and you should be able to join now.

**Reply by Ryuminh** [reply]
thanks worked well i joined the servor :) thanks for your help

**Reply by ashik0121** [reply]
hello i have some issue, i was in the discord channel and i don't know why i'm not in there anymore and i can't join it

**Reply by bxpana** [reply]
@ashik0121 as mentioned in this thread and in other places you have posted, please email us at support@zksync.io

**Reply by ashik0121** [reply]
![Screenshot_2023-12-08-23-32-33-353_com discord](https://github.com/zkSync-Community-Hub/zkync-developers/assets/53506661/acad893e-ca37-4433-985b-675f0b5f8251)

### **I can't join here can you solve it**

**Comment by Ryuminh** [comment]
ok @bxpana thanks for it :)

**Comment by Ryuminh** [comment]
Hello

I tried again but the link Said link is not valid or expired liké this

**Comment by ashik0121** [comment]
> @ashik0121 as mentioned in this thread and in other places you have posted, please email us at [support@zksync.io](mailto:support@zksync.io)

![Screenshot_2023-12-08-23-32-33-353_com discord](https://github.com/zkSync-Community-Hub/zkync-developers/assets/53506661/d01cc02f-1a6c-4e86-bc3a-f31f41afb09a)

**I can't join here can you solve it**
